system/ping/ping.c

```diff
@@ -242,10 +242,17 @@ static void ping_result(FAR const struct ping_result_s *result)
                    result->extra / USEC_PER_MSEC);
             if (result->nreplies > 0)
```

Comment:
let's merge line 249 to here:
```
if (result->nreplies > 0 && priv->tsum > 0)
```

Reply:
In this case we won't print anything if the delay is 0.  Is that ok?
I pushed as suggested but I am having a second guess.

Reply:
Sorry, I forget the print, either approach is fine. Since you encounter this special situation, and should know which one is best choice:).

Reply:
I believe we should keep the print since we had a valid number of replies.  I added it back, please take a look.